vta/src/dpi/module.cc

```diff
@@ -124,10 +124,10 @@ void HostDevice::PushRequest(uint8_t opcode, uint8_t addr, uint32_t value) {
   req_.Push(r);
 }
 
-bool HostDevice::TryPopRequest(HostRequest &r, bool pop) {
-  r.opcode = 0xad;
-  r.addr = 0xad;
-  r.value = 0xbad;
+bool HostDevice::TryPopRequest(HostRequest *r, bool pop) {
```

Comment:
Style consistency with rest of the code: use ```HostRequest* r```

Reply:
Fixed it.